components/header.tsx

```diff
@@ -18,7 +26,24 @@ export default function Header() {
           </a>
         </Link>
       </Flex>
-      <Flex as="nav" sx={{ justifyContent: "center" }}>
+      <Flex
+        as="nav"
+        className={isActive ? "d-flex" : ''}
+        sx={{
+          justifyContent: "center",
+          [MEDIA_QUERY.mobile]: {
+            display: 'none',
+            position: 'fixed',
+            left: 0,
+            right: 0,
+            bottom: 0,
+            backgroundColor: t => t.colors.background,
```

Comment:
you can just use background as string here no need to use the function and you can use bg as shortcode

```
bg: 'background',
```